src/lib/css/nav.scss

```
@@ -62,6 +62,12 @@ nav {
         font-size: .7em;
       }
     }
+
+    .dropdown .dropdown-menu {
```

Una solución potable por ahora, sería agregar la clase `.plex-box` al contenedor del menú (/src/lib/app/app.component.ts:45) y modificar estas reglas así:
```suggestion

.dropdown {
      background: $blue;
      &:hover {
        background: $dark-blue;
      }
      
      .dropdown-menu {
          max-height: 80vh;
          overflow-y: scroll;    
          @include scrollbar();    
      }
    }
```